mqtt-streaming/src/main/scala/org/apache/pekko/stream/connectors/mqtt/streaming/model.scala

```diff
@@ -53,7 +53,8 @@ object ControlPacketType {
   val DISCONNECT = ControlPacketType(14)
   val Reserved2 = ControlPacketType(15)
 }
-final case class ControlPacketType private (underlying: Int) extends AnyVal
+@InternalApi
+final case class ControlPacketType(underlying: Int) extends AnyVal
```

Comment:
private[connectors] ?

Reply:
the code is used in a test that is in the `docs.scaladsl` package - so the private[connectors] is invisible to that package